### PR TITLE
Final Touches

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
### Leaderboard: final touches
- Using plain CSS framework

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -57,7 +75,7 @@ form input {
 }
 
 .leaders:nth-child(even) {
-  background-color: hsl(182, 95%, 91%);
+  background-color: hsl(175, 73%, 89%);
 }
 
 .btn-submit {
```
Comment (V-Blaze) [inline src/style.css:81]: - Your project looks like the provided Design , :+1: :+1: But there are few changes you need to make before going to the next milestone. 

- [ ] This should have been pointed out by the previous reviewers, According to Microverse, you are meant to remove all the placeholder comments on the readme as they are there to guide you as you customize the readme. I suggest you remove all the placeholder comments here. :smile: :small_airplane: 

- - Screenshots for reference:
Selection_012
Selection_013

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @HenaryKhuraijam 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- UI looks great✔
-  Correct Github flow👋 
- All app functionalities works great :heavy_check_mark: 

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (HenaryKhuraijam) [reply src/style.css:81]: Thank you so much for your support. Done

**Review by ademibowale — APPROVED** [review]
Hi @HenaryKhuraijam , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️
- styling fit design design template ✔️
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**